modules/@ergonode/category-trees/src/store/categoryTree/actions.js

```diff
@@ -103,51 +103,44 @@ export default {
             });
             // EXTENDED BEFORE METHOD
 
+            const [
+                categoryTreeData,
+                categoriesData,
+            ] = await Promise.all([
+                get({
+                    $axios: this.app.$axios,
+                    id,
+                }),
+                getAutocomplete({
+                    $axios: this.app.$axios,
+                }),
+            ]);
+
             const {
                 language: userLanguageCode,
             } = rootState.authentication.user;
-            const data = await get({
-                $axios: this.app.$axios,
-                id,
-            });
+
             const {
                 code,
                 name = '',
                 categories,
-            } = data;
+            } = categoryTreeData;
```

Comment:
maybe it would be better:  `} = categoryTreeData || {};` 
fallback for null and undefined